metadata.json

```diff
@@ -57,7 +58,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.5.8 < 7.0.0"
+      "version_requirement": ">= 6.1.0 < 8.0.0"
```

Comment:
This change is not mentioned in the changelog this way.

Reply:
it is, in https://github.com/voxpupuli/puppet-keepalived/pull/234 (which this one includes)